modulefiles/ufs_hera.intel.lua

```diff
@@ -2,40 +2,26 @@ help([[
 loads UFS Model prerequisites for Hera/Intel
 ]])
 
-prepend_path("MODULEPATH", "/contrib/sutils/modulefiles")
-load("sutils")
+prepend_path("MODULEPATH", "/scratch1/NCEPDEV/nems/role.epic/spack-stack/spack-stack-1.4.1/envs/unified-env/install/modulefiles/Core")
+prepend_path("MODULEPATH", "/scratch1/NCEPDEV/jcsda/jedipara/spack-stack/modulefiles")
 
-cmake_ver=os.getenv("cmake_ver") or "3.20.1"
-load(pathJoin("cmake", cmake_ver))
-
-intel_ver=os.getenv("intel_ver") or "2022.1.2"
-load(pathJoin("intel", intel_ver))
-
-impi_ver=os.getenv("impi_ver") or "2022.1.2"
-load(pathJoin("impi", impi_ver))
-
-prepend_path("MODULEPATH", "/scratch1/NCEPDEV/nems/role.epic/hpc-stack/libs/intel-2022.1.2/modulefiles/stack")
+stack_intel_ver=os.getenv("stack_intel_ver") or "2021.5.0"
+load(pathJoin("stack-intel", stack_intel_ver))
 
-hpc_ver=os.getenv("hpc_ver") or "1.2.0"
-load(pathJoin("hpc", hpc_ver))
+stack_impi_ver=os.getenv("stack_impi_ver") or "2021.5.1"
+load(pathJoin("stack-intel-oneapi-mpi", stack_impi_ver))
 
-hpc_intel_ver=os.getenv("hpc_intel_ver") or "2022.1.2"
-load(pathJoin("hpc-intel", hpc_intel_ver))
+stack_python_ver=os.getenv("stack_python_ver") or "3.9.12"
+load(pathJoin("stack-python", stack_python_ver))
 
-hpc_impi_ver=os.getenv("hpc_impi_ver") or "2022.1.2"
-load(pathJoin("hpc-impi", hpc_impi_ver))
-
-scotch_ver=os.getenv("scotch_ver") or "7.0.3"
-load(pathJoin("scotch", scotch_ver))
+cmake_ver=os.getenv("cmake_ver") or "3.23.1"
+load(pathJoin("cmake", cmake_ver))
 
 load("ufs_common")
 
-setenv("CC", "mpiicc")
-setenv("CXX", "mpiicpc")
-setenv("FC", "mpiifort")
+setenv("CC", "icc")
+setenv("CXX", "icpc")
+setenv("FC", "ifort")
 setenv("CMAKE_Platform", "hera.intel")
 
-prepend_path("MODULEPATH", "/scratch2/BMC/ifi/modulefiles")
```

Comment:
@ulmononian May I know why this ifi module is removed? It is needed for generating IFI product in in-line post. @SamuelTrahanNOAA may chime in.

Reply:
Don't remove the ifi module commands.

Reply:
Sorry, I misread this. It is okay to remove them from Jet and Hera. Just don't remove them from Acorn.

Reply:
@SamuelTrahanNOAA Are you meaning  that testing ifi generation in inline post on Acorn would be good enough?

Reply:
I just checked the PR and it does *not* remove the Acorn ifi modulefiles. That is the only platform where the UFS regression tests need to load any ifi modulefiles.

All of the parallels that run IFI do so outside the model, using standalone UPP. The UPP has its own modulefiles. Thus, this won't affect the parallels either.

Reply:
@SamuelTrahanNOAA Got it. Thanks for clarifying.